gcp/service-account-key/mainf.tf

```diff
@@ -0,0 +1,13 @@
+terraform {
+  required_version  = ">=0.12.3"
+}
+
```

Comment:
You are missing a google provider here as well:

![image](https://user-images.githubusercontent.com/46656490/80996686-ec366a00-8e48-11ea-8d39-cdc06f3a3e95.png)

Reply:
Thanks,
added it and verified it works.
233643c